configs/11.0/build.mk

```diff
@@ -1 +0,0 @@
-../10.0/build.mk
```

Comment:
It's not your patch fault but it will break:
$ readlink configs/next/build.mk 
../10.0/build.mk

It should be ../11.0/build.mk